wordstats/word_distance.py

```diff
@@ -0,0 +1,132 @@
+import codecs
+import configparser
+from .cognate_files_path import *
+
+class WordDistance(object):
+    def __init__(self):
```

Comment:
a class comment would help.
also explaining the meaning of the instance variables: replace_distance, add_distance, and threshold.